.env.example

```diff
@@ -26,4 +26,5 @@ NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
I had some merge conflicts around these lines, so I might not have fixed them correctly. Is it better to use dev urls in the example file or staging?